### PR TITLE
iterate setup/test files

### PR DESCRIPTION
**Pull request by mortelli**
This PR adds files to ease the boot up process and execute the test suite.

### CODE REVIEW

File: migrations/2_deploy_contracts.js [hunk 1]
```diff
@@ -23,10 +22,33 @@ module.exports = async function (deployer) {
...
   await deployer.deploy(CustomSmartWalletDeployVerifier, CustomSmartWalletFactory.address)
-  await deployer.deploy(CustomSmartWalletRelayVerifier, CustomSmartWalletFactory.address)
+  await deployer.deploy(RelayVerifier, CustomSmartWalletFactory.address)
+
+  const customSmartWalletRelayVerifierAddress = RelayVerifier.address;
+
   await deployer.deploy(SampleRecipient)
-  await deployer.deploy(TestToken)
+
+  console.log('Done! Summary:')
```
Comment (JONAF2103) [inline migrations/2_deploy_contracts.js:37]: Can we change this to something like:

```suggestion
console.log('Done! Summary:')

  console.log('|===================================|============================================|')
  console.log('| Contract                          | Address                                    |')
  console.log('|===================================|============================================|')
  console.log(`| Penalizer                         | ${Penalizer.address} |`)
  console.log(`| RelayHub                          | ${RelayHub.address} |`)
  console.log(`| Smart Wallet Contracts ========================================================|`)
  console.log(`| SmartWallet                       | ${SmartWallet.address} |`)
  console.log(`| SmartWalletFactory                | ${SmartWalletFactory.address} |`)
  console.log(`| SmartWalletDeployVerifier         | ${DeployVerifier.address} |`)
  console.log(`| SmartWalletRelayVerifier          | ${smartWalletRelayVerifierAddress} |`)
  console.log(`| Custom Smart Wallet Contracts =================================================|`)
  console.log(`| CustomSmartWallet                 | ${CustomSmartWallet.address} |`)
  console.log(`| CustomSmartWalletFactory          | ${CustomSmartWalletFactory.address} |`)
  console.log(`| CustomSmartWalletDeployVerifier   | ${CustomSmartWalletDeployVerifier.address} |`)
  console.log(`| CustomSmartWalletRelayVerifier    | ${customSmartWalletRelayVerifierAddress} |`)
  console.log(`| Testing Contracts =============================================================|`)
  console.log(`| SampleRecipient                   | ${SampleRecipient.address} |`)
  console.log(`| TestToken                         | ${TestToken.address} |`)
  console.log('|===================================|============================================|\n')

```

**Review by JONAF2103 — APPROVED** [review]
LGTM, we should improve some of the scripts in the future but it's ok to have them right now in the main branch, at least to accelerate the testing and development process.

File: migrations/2_deploy_contracts.js (hunk 1 above)
Comment (mortelli) [reply migrations/2_deploy_contracts.js:37]: done :+1:

**Review by mauricioirace — APPROVED** [review]
LGTM!

**Pull request merged**